### PR TITLE
More aliases.

### PR DESCRIPTION
**Pull request by samscott89**

### CODE REVIEW

File: docs/content/python/guides/roles/sqlalchemy/_index.md [hunk 1]
```diff
@@ -4,5 +4,8 @@ weight: 1
 description: >
     Use Oso with SQLAlchemy to handle all roles data management,
     expose roles to end users, and enforce authorization.
+aliases:
+ - ../../new-roles.html
```
Comment (samscott89) [inline docs/content/python/guides/roles/sqlalchemy/_index.md:8]: We link people to `/python/new-roles.html` in an error message.

**Review by gj — APPROVED** [review]

File: docs/content/any/guides/roles/_index.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ title: "Build Role-Based Access Control"
 weight: 1
 showContentForAnyLanguage: true
 aliases:
-  - /python/getting-started/roles/builtin_roles.html
+ - /python/getting-started/roles/builtin_roles.html
+ - new-roles.html
```
Comment (samscott89) [inline docs/content/any/guides/roles/_index.md:7]: e.g. `/python/guides/new-roles.html` redirects to `/python/guides/roles.html`

File: docs/content/python/guides/roles/sqlalchemy/_index.md (hunk 1 above)
Comment (samscott89) [reply docs/content/python/guides/roles/sqlalchemy/_index.md:8]: e.g. `/python/new-roles.html` redirects to `/python/guides/roles/sqlalchemy.html`

File: docs/content/any/guides/roles/_index.md (hunk 2 above)
Comment (samscott89) [reply docs/content/any/guides/roles/_index.md:7]: or even `/guides/new-roles.html`

**Pull request merged**